tests/NuGet.Services.AzureSearch.Tests/Support/SerializationUtilities.cs

```diff
@@ -15,14 +21,26 @@ public class SerializationUtilities
         public static async Task<string> SerializeToJsonAsync<T>(T obj) where T : class
         {
             using (var testHandler = new TestHttpClientHandler())
-            using (var serviceClient = new SearchServiceClient(
-                "unit-test-service",
-                new SearchCredentials("unit-test-api-key"),
-                testHandler))
             {
-                var indexClient = serviceClient.Indexes.GetClient("unit-test-index");
-                await indexClient.Documents.IndexAsync(IndexBatch.Upload(new[] { obj }));
-                return testHandler.LastRequestBody;
+                var indexClient = new SearchClient(
+                   new Uri("https://unit-test-service/"),
```

Comment:
Consider adding a `.test` top level domain so the URL is guaranteed to not be real. I don't feel strongly about this, feel free to skip

```suggestion
                   new Uri("https://unit-test-service/"),
```